internal/backend/gs/config.go

```diff
@@ -16,7 +16,8 @@ type Config struct {
 	Bucket    string
 	Prefix    string
 
-	Connections uint `option:"connections" help:"set a limit for the number of concurrent connections (default: 20)"`
+	Connections uint   `option:"connections" help:"set a limit for the number of concurrent connections (default: 20)"`
```

Comment:
As your PR already modifies this line, please fix the comment to state the actual default of "5" connections.